native/src/routes/settings/components/SettingItem.js

```diff
@@ -1,24 +1,29 @@
 // @flow
```

Comment:
Refactoring only, moved some functionality from `Settings` here: badges and switches